app/assets/javascripts/modules/external_users/claims/CaseTypeCtrl.js

```diff
@@ -39,7 +39,7 @@ moj.Modules.CaseTypeCtrl = {
   bindEvents: function () {
     var self = this;
 
-    $.subscribe('/onConfirm/claim_case_type_id-select/', function (e, data) {
+    $.subscribe('/onConfirm/case_type-select/', function (e, data) {
```

Comment:
Fix show/hide of date fields depending on case type selection (Trial, Re-trial etc.)